readme.md

```diff
@@ -152,13 +155,13 @@ You can also choose to use a named function instead:
 
 ```js
 test(function name(t) {
-	t.end();
+	t.pass();
 });
 ```
 
 ### Planned assertions
 
-Planned assertions are useful for being able to assert that all async actions happened. It also comes with the benefit of not having to manually end the test.
+Planned assertions are useful for being able to assert that all async actions happened. 
```

Comment:
Can you add something about that it's also useful to make sure not too many assertions were run?